Sources/Sentry/SentrySdkInfo.m

```diff
@@ -44,6 +44,25 @@ - (instancetype)initWithDict:(NSDictionary *)dict
     return [self initWithName:name andVersion:version];
```

Comment:
I wanted to reuse the new method with sth like:

```objective-c
return [self initWithDict:dict orDefaults:nil];
```

but that would require the arg to be nullable - do you want me to make the change?